extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -160,7 +177,9 @@ export class AmpSidebar extends AMP.BaseElement {
         }, ANIMATION_TIMEOUT);
       });
     });
-    this.getHistory_().push(this.close_.bind(this)).then(historyId => {
+    this.getHistory_().push(() => {
+      this.close_();
```

Comment:
During myinitiall experiments I couldn't fix with a small timeout. I didn't investigate this enough bit wanted to see if you might have an idea why that might be happening. 

I'll do some more work on this and see if I can use the promises for this.